packages/paypal-commerce-integration/src/paypal-commerce/create-paypal-commerce-button-strategy.ts

```diff
@@ -16,12 +20,14 @@ const createPayPalCommerceButtonStrategy: CheckoutButtonStrategyFactory<
 > = (paymentIntegrationService) => {
     const { getHost } = paymentIntegrationService.getState();
 
-    return new PayPalCommerceButtonStrategy(
+    const paypalCommerceCommon = new PayPalCommerceCommon(
```

Comment:
I like the whole idea very much, but I would recommend to change naming a bit. 
From PayPalCommerceCommon to something like PayPalCommerceAPI, PayPalCommerceToolkit,  PayPalCommerceSDK.